src/components/CLI/CLI.tsx

```diff
@@ -42,6 +61,33 @@ export default function Cli({ decreaseCommandsLeft }: CliProps) {
   const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
     if (e.key === 'Enter') {
       handleCommandWrapper(e);
+      setCommandHistory(prev => [...prev, command]);
```

Comment:
When user presses Enter on an empty input, the command should not be added to the history correct?

Reply:
correct, I'll fix it